packages/collector/test/tracing/cloud/aws/s3/app.js

```diff
@@ -132,6 +135,7 @@ const S3Api = {
 
           promise
             .then(data => {
+              log(` /${operation}/${method} - received data from AWS SDK`);
```

Comment:
There's an extra space at the start of string. Accidental or deliberate?

Reply:
Accidental. thanks!